lms/djangoapps/discussion/views.py

```diff
@@ -810,7 +810,11 @@ def render_to_fragment(  # lint-amnesty, pylint: disable=arguments-differ
         course_key = CourseKey.from_string(course_id)
         # Force using the legacy view if a user profile is requested or the URL contains a specific topic or thread
         force_legacy_view = (profile_page_context or thread_id or discussion_id)
-        is_educator_or_staff = is_course_staff(course_key, request.user) or GlobalStaff().has_user(request.user)
+        is_educator_or_staff = (
```

Comment:
This is not a comment about your code, but apparently this variable is never used.